ansible/roles/collectory/tasks/main.yml

```diff
@@ -44,10 +67,18 @@
     - properties
     - collectory
 
-- name: set data ownership [all data is owned by tomcat]
-  file: path={{ collectory_data_dir }} owner={{ tomcat_user }}  group={{ tomcat_user }}  recurse=true
-  notify: 
-    - restart tomcat  
+#- include_tasks: ../../tomcat_deploy/tasks/main.yml
```

Comment:
Remove this unused, commented out, `tomcat_deploy` section.

Reply:
removed